using_images/s2i_images/dot_net_core.adoc

```diff
@@ -0,0 +1,131 @@
+running .NET applications without having to rely solely on Windows Server.
+* Access to many of the popular development frameworks such as .NET, Java, Ruby,
+and Python from within {product-title}.
+
+[[dot-net-core-supported-versions]]
+== Supported Versions
+
+* .NET Core version 1.0
+** Supported on Red Hat Enterprise Linux (RHEL) 7 {product-title} versions 3.3 and later
+
+ifdef::openshift-enterprise[]
+[[dot-net-core-installing-images]]
+== Images
+
+Image stream definitions for the .NET Core on RHEL S2I image are now added
+during {product-title} installations.
+endif::openshift-enterprise[]
+
+[[dot-net-core-configuration]]
+== Configuration
+
+The .NET Core image currently does not support any environment variables. No
+configuration is necessary.
+
+ifdef::openshift-enterprise[]
+[[dot-net-quickly-deploy-applications]]
+== Quickly Deploying Applications from .NET Core Source
+
+As of {product-title} version 3.3, an image can be used to build an application
+by running `oc new-app` against a sample repository:
+
+----
+$ oc new-app https://github.com/redhat-developer/s2i-dotnetcore --context-dir=1.0/test/asp-net-hello-world
+----
+
+[IMPORTANT]
+====
+The .NET image stream must first be installed. If you ran a standard
+installation, the image stream will be present.
+====
+endif::openshift-enterprise[]
+
+[[dot-net-manually-deploying]]
+== Manually Deploying Applications
+
+[[dot-net-compile-sources]]
+=== Compile Sources
+
+You must compile the sources before you can run your application.
+
+. Restore the dependencies and tools of a project:
++
+----
+$ dotnet restore
+----
+
+. Build the project and all of its dependencies:
++
+----
+$ dotnet build -c Release
+----
+
+[[dot-net-core-run-your-application]]
+=== Run Your Application
+
+To run your application:
+
+----
+$ dotnet run
+----
+
+[[dot-net-publishing-options]]
+=== Deploy Your Application
+
+{nbsp}.NET Core can produce a fat binary, including compiled user code and the
+{nbsp}.NET virtual machine (VM). This can be used to build a smaller VM for
+deployment.
+
+[NOTE]
+====
+The fat binary. which includes both the built source and the .NET Core runtime,
+is not portable across architectures. It is a native binary. The version built
+for Red Hat Enterprise Linux (RHEL), for example, will not run on Microsoft
+Windows.
+====
+
+There are
+link:https://docs.microsoft.com/en-us/dotnet/articles/core/deploying/[two ways of publishing applications]:
+
+. Build the source code and produce an artifact that relies on the .NET Core
+runtime being present on the system where the code is executed. Use the `dotnet
+publish` command.
+
+. Build the source code and bundle in any dependencies, including the .NET Core
+ runtime, to produce an application that runs without needing a .NET
+ Core runtime on the system where it is being deployed.
+.. Ensure the runtime (*rhel.7-x64*) is present in the *_project.json_*.
+
+.. Run:
++
+----
+$ dotnet publish <other args> -r rhel.7-x64
+----
```

Comment:
i don't understand this whole "deploying manually" section.  If the intent is to explain to users how to build their application locally and then package it into an image and run that image, this section stops short of actually explaining how to package their compiled application into an image or get it deployed.

As it is, it just reads like a tutorial on how to locally build/package a dotnet app, which isn't relevant to either openshift or the dotnet image..so what is the intent?